cdap-docs/_common/common-build.sh

```diff
@@ -14,19 +14,12 @@
 # License for the specific language governing permissions and limitations under
 # the License.
   
-# Build script for docs
-# Builds the docs (all except javadocs and PDFs) from the .rst source files using Sphinx
-# Builds the javadocs and copies them into place
-# Zips everything up so it can be staged
-# REST PDF is built as a separate target and checked in, as it is only used in SDK and not website
-# Target for building the SDK
-# Targets for both a limited and complete set of javadocs
-# Targets not included in usage are intended for internal usage by script
+# Common code for Build script for docs
+# Not called directly; included in either the main build.sh or the individual manual's build.sh
```

Comment:
This isn't visible in the PR, but you shouldn't have a shebang header in a shell file that isn't meant to be executed.